### DISCUSSION TITLE
GitHub logo only supplied in proprietary Adobe formats (.ai, .eps)

### DISCUSSION THREAD

**Discussion by kmturley** [community/community#39875] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

On this page you can download GitHub logo assets:
https://github.com/logos

However the vector formats are both proprietary Adobe formats:

- .ai = https://en.wikipedia.org/wiki/Adobe_Illustrator_Artwork
- .eps = https://en.wikipedia.org/wiki/Encapsulated_PostScript

For accessibility, logos should be distributed in an open standard such as:

- .svg = https://en.wikipedia.org/wiki/Scalable_Vector_Graphics

An example of a company doing this is Discord:
https://discord.com/branding

Thanks!

**Comment by ossanna16** [comment]
Hi there @kmturley and welcome to our community! Thank you for the feedback!
